### PR TITLE
[wintoast] Update to 1.3.0

### PR DESCRIPTION
**Pull request by nickdademo**
- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [x] SHA512s are updated for each updated download
- [ ] ~~The "supports" clause reflects platforms that may be fixed by this new version~~
- [ ] ~~Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.~~
- [ ] ~~Any patches that are no longer applied are deleted from the port's directory.~~
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

**Comment by nickdademo** [comment]
@microsoft-github-policy-service agree company="DUERR NDT GmbH & Co. KG"

**Comment by nickdademo** [comment]
Unfortunately, the upstream repo does not support CMake install - is this the best way to handle this?

**Comment by Cheney-W** [comment]
The `CMakeLists.txt` provided by the upstream and the one provided by vcpkg do not differ in terms of target construction, but lack support for usage. Therefore, I believe there is no need to delete the `CMakeLists.txt` provided by vcpkg, as it would render the current usage unavailable and require additional processing (as seen in your modifications in the `portfile.cmake`).

File: ports/wintoast/portfile.cmake [hunk 1]
```diff
@@ -1,25 +1,26 @@
 vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
+vcpkg_minimum_required(VERSION 2022-10-12) # for ${VERSION}
```
Comment (Cheney-W) [inline ports/wintoast/portfile.cmake:2]:
```suggestion
```
Now you can directly use `${VERSION}`, and this line is no longer needed.

File: ports/wintoast/vcpkg.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "wintoast",
-  "version": "1.2.0",
+  "version": "1.3.0",
   "port-version": 2,
```
Comment (Cheney-W) [inline ports/wintoast/vcpkg.json:4]:
```suggestion
```
The version has been updated, please remove the `port-version` item.

**Comment by dg0yt** [comment]
> Unfortunately, the upstream repo does not support CMake install - is this the best way to handle this?

It should be simple to add as a patch.

**Comment by nickdademo** [comment]
> The `CMakeLists.txt` provided by the upstream and the one provided by vcpkg do not differ in terms of target construction, but lack support for usage. Therefore, I believe there is no need to delete the `CMakeLists.txt` provided by vcpkg, as it would render the current usage unavailable and require additional processing (as seen in your modifications in the `portfile.cmake`).

Do you know of any good existing port example where the upstream repo does not create/install a CMake config?

File: ports/wintoast/Config.cmake.in [hunk 3]
```diff
@@ -0,0 +1,18 @@
+if(NOT TARGET wintoast)
+    add_library(wintoast STATIC IMPORTED)
+
+    find_path(WinToast_INCLUDE_DIR NAMES wintoastlib.h)
```
Comment (dg0yt) [inline ports/wintoast/Config.cmake.in:4]: No `find_path` needed: you know the exact relative location at installation time.

File: ports/wintoast/Config.cmake.in [hunk 4]
```diff
@@ -0,0 +1,18 @@
+if(NOT TARGET wintoast)
+    add_library(wintoast STATIC IMPORTED)
```
Comment (dg0yt) [inline ports/wintoast/Config.cmake.in:2]:
```suggestion
    add_library(wintoast UNKNOWN IMPORTED)
```

**Review by dg0yt — COMMENTED** [review]
If you want to add vcpkg-only cmake config, implement CMake package `unofficial-wintoast`and CMake target `unofficial::wintoast::wintoast`.
https://learn.microsoft.com/en-us/vcpkg/contributing/maintainer-guide#add-cmake-exports-in-an-unofficial--namespace

I think the current config is broken for release-only builds.

**Review by Cheney-W — APPROVED** [review]

**Comment by Cheney-W** [comment]
Usage passed with following triplets:
x64-windows 
x64-windows-static

**Pull request merged**